commands/help.js

```diff
@@ -0,0 +1,36 @@
+const { SlashCommandBuilder } = require('discord.js');
+
+module.exports = {
+  data: new SlashCommandBuilder()
+    .setName('help')
+    .setDescription('Information'),
+  async execute(interaction) {
+    const user = interaction.user;
+
+    const message = `
```

Comment:
Its definitely not too too long but might look cleaner if this was also in assets folder / renamed folder, same as other one